pkg/helm/controller/reconcile.go

```diff
@@ -240,7 +240,8 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 			r.EventRecorder.Eventf(o, "Warning", "OverrideValuesInUse",
 				"Chart value %q overridden to %q by operator's watches.yaml", k, v)
 		}
-		previousRelease, updatedRelease, err := manager.UpdateRelease(context.TODO())
+		force := o.GetAnnotations()["operator-sdk.coreos.com/force"] == "true"
```

Comment:
We have a new domain `operator-framework.io` so let's use that here.
```suggestion
		force := o.GetAnnotations()["helm.sdk.operator-framework.io/force"] == "true"
```